lib/PuppeteerSharp.Nunit/PuppeteerTestAttribute.cs

```diff
@@ -170,14 +171,24 @@ private static TestExpectation.TestExpectationPlatform GetCurrentExpectationPlat
         private static TestExpectation[] GetUpstreamExpectations() =>
             _upstreamExpectations ??= LoadExpectationsFromResource("PuppeteerSharp.Nunit.TestExpectations.TestExpectations.upstream.json");
 
+        private static JsonSerializerOptions DefaultJsonSerializerOptions =>
```

Comment:
Needs to be a field to be a cached instance just like
https://github.com/hardkoded/puppeteer-sharp/pull/2713/files#diff-af9c020cb3f1613906500f6b77a8dd95e2f19221ed17509db28c47a5ff5dd55dR10
Since this is currently just a shortcut for `DefaultJsonSerializerOptions { get { return new() { ... }; } }` which would return a new instance every time.

```suggestion
        private static readonly JsonSerializerOptions DefaultJsonSerializerOptions =
```

Reply:
oh yeah, good catch